Classes/Service/ContentService.php

```diff
@@ -225,8 +226,6 @@ public function moveRecord(array &$row, &$relativeTo, $parameters, DataHandler $
 			} elseif (0 < (integer) $relativeTo) {
 				// moving to first position in colPos, means that $relativeTo is the pid of the containing page
 				$row['sorting'] = $tceMain->getSortNumber('tt_content', 0, $relativeTo);
-				$row['tx_flux_parent'] = NULL;
```

Comment:
But removing those lines causes the element to not be moved if you drag it from inside a nested container to the top column position.

References:

https://github.com/FluidTYPO3/flux/commit/fab20a070409112b04c38c94c8b7ef69352fa72f
https://github.com/FluidTYPO3/flux/commit/7e20161288e6a830565e288bab91e908ca196ea3
https://github.com/FluidTYPO3/flux/commit/c0864b4f02ccd7382146e6f15737bf3576d2ed0e

Enabling the first feature means you can sort elements using the arrows and face the problem you have. The second one is a patch that fixes the drag and drop. Last is the original patch that made child records invisible and unsortable in list view (but sortable using the Relations tab of the parent).

I'm pretty sure at this point that we must have these lines and somehow learn to accept the problem of list view sorting child elements using the arrow icons...